packages/core/src/customer/customer-strategy-selector.ts

```diff
@@ -152,6 +153,16 @@ export function createCustomerStrategySelectorFactory(): CustomerStrategySelecto
         },
     );
 
+    const isLoadingWalletButton = createSelector(
+        (state: CustomerStrategyState) => state,
+        (state) => (methodId: string) => {
+            return !(
```

Comment:
How come we can't just use `isInitializing`? If a button fails to load, it doesn't mean it's still loading.

Reply:
The outcome of `isLoading` will be affected by their different definitions.

- `isInitializing` is `true` if a button is actually loading.
- `isLoadingWalletButton` is `true` if a button is not initialized or failed.


There is always a tiny moment where checkout does not load anything, making `isLoading` equal `false`.  The loading skeleton will thereafter be on and off again and again.
\* screenshot of `console.log(methodIds.filter((methodId) => isInitializingCustomer(methodId); }))`.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/88361607/223001006-8f60581a-666e-40b3-8744-d3f0acdef8fa.png">

`isLoadingWalletButton` can provide the best results of deciding when to show/hide loading skeletons.
\* screenshot of `console.log(methodIds.filter((methodId) => isLoadingWalletbutton(methodId); }))`.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/88361607/223001817-cbfd8329-fd68-4e27-b2cb-6f28e77e2964.png">

I just want to avoid switching the skeleton on and off. Is there another way we can do it?